substrate/bin/node/executor/src/lib.rs

```diff
@@ -15,26 +15,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! A `CodeExecutor` specialization which uses natively compiled runtime when the wasm to be
-//! executed is equivalent to the natively compiled code.
+//! Provides executor related types intended to be used across Substrate node.
 
-pub use sc_executor::NativeElseWasmExecutor;
+/// Host functions required for kitchensink runtime and Substrate node.
+pub type HostFunctions =
```

Comment:
This entire crate should be removed.

Reply:
I kept the crate because of benches and tests: https://github.com/paritytech/polkadot-sdk/tree/master/substrate/bin/node/executor.

Should they be removed too?

Reply:
They can just be moved to the `node` crate itself.

Reply:
Types moved to `staging-node-cli` in 990c05e.